pkg/api/controller.go

```diff
@@ -187,7 +187,12 @@ func (c *Controller) Login(w http.ResponseWriter, r *http.Request, body LoginJSO
 
 	// user.Username will be different from username/access_key_id on
 	// LDAP login.  Use the stored value.
-	tokenString, err := GenerateJWTLogin(secret, user.ID, loginTime, expires)
+	n, err := auth.UserIDToInt64(user.ID)
```

Comment:
We should already advance to use the KV models which have a string ID and modify all the necessary code. We should use the DB models only in the transition to write/read from DB